CHANGELOG.md

```diff
@@ -1,7 +1,12 @@
-## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.1.1...master)
+## [Ongoing](https://github.com/pbrisbin/downgrade/compare/v8.2.0...master)
 
 None
 
+## [v8.2.0](https://github.com/pbrisbin/downgrade/compare/v8.1.1...v8.2.0)
+
+- [NEW] Pacman cache directory accumulation to support multiple cache locations
```

Comment:
Just noting here that when I read "cache directory accumulation" I keep thinking it means the cache is accumulating files, not that the options accumulate. Not sure if it's worth rewording in the CHANGELOG?

> - [NEW] Now accepts multiple cache directory options

?

Reply:
Aha yes. Will change.